docs/src/pages/en/get-started.mdx

```diff
@@ -112,30 +122,17 @@ Use HTML to add or remove any of the controls. Then you can use CSS to style the
-| `<media-seek-backward-button>` | Jump back `n` seconds in the media (default 30). ([docs](./components/media-seek-backward-button))                                                                                                                                                                                |
-| `<media-seek-forward-button>`  | Jump ahead `n` seconds in the media (default 30). ([docs](./components/media-seek-forward-button))                                                                                                                                                                                |
-| `<media-captions-button>`      | Show/disable captions (if no captions are available, will fallback to subtitles by default unless `nosubtitlesfallback` attribute is set). ([docs](./components/media-captions-button))                                                                                         |
-| `<media-poster-image>`         | Show a poster image that's displayed until media begins playing for the first time. Optionally also accepts a `placeholdersrc` attribute that can be used for content that immediately loads, such as an inlined, low-resolution image. ([docs](./components/media-poster-image) |
-| `<media-airplay-button>`       | Bring up the AirPlay menu to select/deselect AirPlay playback (Safari only). ([docs](./components/media-airplay-button))                                                                                                                                                          |
-| `<media-loading-indicator>`    | Show when your media content is loading/buffering. ([docs](./components/media-loading-indicator))                                                                                                                                                                                 |
-| More to come                   | Requests and contributions welcome                                                                                                                                                                                                                                             |
-
-
-## Compatible players
-
-Media Chrome will work with any HTML element that exposes the same API as HTML media elements (`<video>` and `<audio>`). See the guide for the [media slot](/en/media-slot) to learn more about how media elements work with the media slot.
+> Unlike the video element we didn't specify a "slot". This just means that the control will be placed into the default slot, which is reserved for controls.
+
+## Taking it further
+
+That's all it takes to start building a custom media player. If you'd like to dive deeper into customization you can check out these resources
+
+- [Styling](/en/styling)
+- [Theming](/en/themes)
+
+## Compatible media elements
```

Comment:
I guess once we get started on the individual element pages, we can probably move this section into an overview page of that section and then update the link above to point there?

Reply:
Yeah this deserves to be a full section on it's own. We can replace with a link out when it's available.